web/src/components/pages/dashboard/dashboard.tsx

```diff
@@ -224,7 +224,7 @@ const ChallengeBar = ({ isNarrow, setShowInviteModal }: ChallengeBarProps) => {
   const [points, setAllPoints] = useState({ user: 0, team: 0 });
 
   useEffect(() => {
-    api.fetchChallengePoints().then(setAllPoints);
+    api.fetchChallengePoints().then(setAllPoints); // TODO: it shouldn't even try to fetch any challenge data in useEffect if not enrolled
   }, []);
```

Comment:
@phirework we'll see what happens on staging, but this line gives me pause. Passing `[]` into useEffect means it's only going to run once per mount. If the first attempt fails because the enrollment isn't there yet, it won't try again when `user` changes.

We may need to figure out _why_ `user` is updating asynchronously and hook into that flow.